frontend/src/app/testResults/mocks/queries.mock.tsx

```diff
@@ -33,20 +32,7 @@ import resultForCorrection from "./resultForCorrection";
 export const mocks = [
   {
     request: {
-      query: GetResultsCountByFacilityDocument,
```

Comment:
These queries are obsolete since we're including the info in the call to the backend via the `totalElements` field